### DISCUSSION TITLE
Preferred method to move a project to new team

### DISCUSSION THREAD

**Discussion by unitof** [vercel/vercel#4388] [General] [answered]
I started a project on my personal Now account, and now would like to move it to a Team account.

I know I cannot move the existing deployment history, but I would like all deploys from now on to go be under the new Team scope.

However Now completely ignores the `--scope` flag when I deploy, instead just deploying to my personal account where all deployments currently are.

I do not have any `scope` property defined in `now.json`.

1. Is this (the ignoring of `--scope`) a bug I should report to Issues? Or by design?
2. What's the preferred method to switch my project's team for future deploys?

Should I add a `scope` to `now.json`?
Or should I delete `/.now` to force `now` to set it up as a new project next deploy?

**Comment by unitof** [comment]
Adding a `scope` to `now.json` also gets ignored, and continues deploying to my personal account anyway.

So I'm gonna delete `/.now` so it deploys fresh, but still would like to know if that's the _best_ way to do this.

**Comment by paulogdm** [comment, marked as answer]
Yes, the fact that the CLI is ignoring the `--scope` flag is by design due to project linking. The ideal approach would be to remove `.now/` or `.vercel/` in order to unlink the project. After that, you can simply use `vercel` to link it again.
